testing/acsccid/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+pkgname=acsccid
+pkgver=1.1.4
+pkgrel=0
+pkgdesc="PCSC driver for ACS CCID smart card reader"
+url="https://github.com/acshk/acsccid"
+arch="all"
+license="LGPL"
+depends="pcsc-lite"
+makedepends="pcsc-lite-dev libusb-dev perl flex"
+options="!check"
```

Comment:
Please write a reason (as a comment on this line) why check is disabled (upstream does not provide tests?).